Directory.Build.targets

```diff
@@ -98,6 +98,35 @@
 
   </ItemGroup>
 
+  <!-- Copied from https://github.com/dotnet/arcade/blob/9d0fd805448082c8d55e2434607b481bca70a146/src/Microsoft.DotNet.Arcade.Sdk/tools/RepositoryInfo.targets#L12-L38 -->
+  <Target Name="_TranslateAzureDevOpsUrlToGitHubUrl"
+          DependsOnTargets="$(SourceControlManagerUrlTranslationTargets)"
+          BeforeTargets="SourceControlManagerPublishTranslatedUrls">
```

Comment:
Where is SourceControlManagerPublishTranslatedUrls? Part of sourcelink?

Reply:
Yup - https://github.com/dotnet/sourcelink/blob/23818e7e498e5e9b3b7f585fb8ac887c3077706d/src/SourceLink.Common/build/InitializeSourceControlInformation.targets#L28